src/cql/upstream-external/NidaQsToUsAuditLogicLibrary.cql

```diff
@@ -0,0 +1,755 @@
+/*  Library: Alcohol Screening and Brief Intervention (ASBI) Clinical Decision Support (CDS): 
+NIDA QS -> USAUDIT Logic Library
+    Author: CMS Alliance to Modernize Healthcare, operated by THE MITRE Corporation.
+
+    (C) 2021 The MITRE Corporation. All Rights Reserved. Approved for Public Release: 20-0458. 
+    Distribution Unlimited.
+
+    Unless otherwise noted, this work is available under an Apache 2.0 license. 
+    It was produced by the MITRE Corporation for the National Center on Birth Defects 
+    and Developmental Disabilities, Centers for Disease Control and Prevention in 
+    accordance with the Statement of Work, contract number 75FCMC18D0047, task order 
+    number 75D30119F05691.
+
+    References to and reproductions of the AUDIT alcohol screening instrument are made 
+    by permission from the World Health Organization (WHO). The WHO does not endorse 
+    this project, does not provide any warranty, and does not assume any liability 
+    for its use. For further information, please see:
+
+    Alcohol Use Disorders Identification Test - Guidelines for Use in Primary Care, 
+    Second Edition. Geneva, World Health Organization, 2001.
+
+    AUDIT (C) World Health Organization 2001
+
+    https://www.who.int/substance_abuse/activities/sbi/en/
+*/
+library NidaQs2UsAuditLogicLibrary version '1.1.0'
+/* Change Log
+- Version 1.1.0: Moving from FHIR 4.0.0 to 4.0.1 and CQL 1.4 to 1.5.
+*/
+
+using FHIR version '4.0.1'
+
+include "FHIRHelpers" version '4.0.1' called FHIRHelpers
```

Comment:
@mcjustin  Since this CQL file is referencing the FHIRHelpers library, we probably should add the CQL source file for the FHIRHelpers library as well.  I think the source CQL for the FHIRHelpers library is [here](https://github.com/cqframework/clinical_quality_language/blob/master/Src/java/quick/src/main/resources/org/hl7/fhir/FHIRHelpers-4.0.1.cql).   I see mention of it [here](https://build.fhir.org/ig/WorldHealthOrganization/smart-anc/Library-FHIRHelpers.html) as well.

Reply:
@ivan-c  would you prefer that an environment config pull this in from its maintained repo dynamically, or should we put a copy in our repo?
https://raw.githubusercontent.com/cqframework/clinical_quality_language/master/Src/java/quick/src/main/resources/org/hl7/fhir/FHIRHelpers-4.0.1.cql

Reply:
Ah, never mind, I should have read @achen2401's message more closely and looked at the code. Adding FHIRHelpers-4.0.1.cql now.

Reply:
> Since this CQL file is referencing the FHIRHelpers library, we probably should add the CQL source file for the FHIRHelpers library as well


It might be nice if we can avoid "vendoring" this CQL library. I just learned the term, but [vendoring means to download and commit source code of a library to your project](https://stackoverflow.com/questions/26217488/what-is-vendoring).

@achen2401 Is there a way to reference a CQL library via permanent URL and have it download during the CQL -> ELM "build" process?

Reply:
hmm, I am not certain.  I have only tried with files on the local file system.  It is not clear from the [repo](https://github.com/cqframework/cql-execution) if that is possible.